src/app/custom-styles.css

```diff
@@ -23,7 +24,23 @@ body {
   background: var(--background);
   font-family: 'Source Sans Pro', sans-serif;
 }
-
+.main-content {
+  background: var(--background);
+}
+.image-card {
+  background: var(--card-background);
+  border-radius: 2.5rem 0rem 2.5rem 0rem;
+  box-shadow: 4px 4px 8px rgba(0, 0, 0, 0.1);
+}
+.image-card .text-box {
+  padding: 2.5rem;
+}
+.image-box {
+  border-radius: 2.5rem 0rem 0rem 0rem;
+  img {
+    border-radius: 2.5rem 0rem 0rem 0rem;
+  }
+}
```

Comment:
Do we want to throw this styling into a `styles.scss` that lives next to `ImageCard.tsx` under `components/ImageCard`?

Reply:
Done!